app/integration_action.go

```diff
@@ -228,16 +228,18 @@ func (a *App) DoActionRequest(rawURL string, body []byte) (*http.Response, *mode
 }
 
 func (a *App) OpenInteractiveDialog(request model.OpenDialogRequest) *model.AppError {
-	clientTriggerId, userId, err := request.DecodeAndVerifyTriggerId(a.AsymmetricSigningKey())
-	if err != nil {
-		return err
+	if request.UserId == "" {
```

Comment:
Would you please add a comment explaining what is going on?

Reply:
Done.  Happy to change the wording if something is better.